.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -0,0 +1,137 @@
+import cimodel.lib.miniutils as miniutils
+
+
+NON_PR_BRANCH_LIST = [
+    "master",
+    r"/ci-all\/.*/",
+    r"/release\/.*/",
+]
+
+
+class WindowJob:
+    def __init__(self,
+                 test_index,
+                 vscode_spec,
```

Comment:
Since this is python-3 only can we use some and define vscode_spec and cuda_version enums?

Reply:
let's save this for a later revision